OpenEdXMobile/res/layout/row_download_list.xml

```diff
@@ -67,8 +62,7 @@
             android:layout_marginBottom="@dimen/widget_margin_half"
             android:layout_marginTop="@dimen/widget_margin_half"
             android:max="100"
-            android:progress="40"
-            tools:targetApi="17" />
+            android:progress="40" />
```

Comment:
Can we change it to `tools:progress="40"`

Reply:
done.